W1-M3-branches-and-loops-local/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+//ExploreIf();
+
+void ExploreIf()

```

Comment:
As we work with other team members, one of our conventions is to use [XML Code Comments](https://docs.microsoft.com/en-us/visualstudio/ide/reference/generate-xml-documentation-comments?view=vs-2022). This not only provides useful information when looking at source code but also gives us in-editor insight when using a class, method, property or event. Please implement XML Code comments on every method, property and event.